Templates/FaceLocator-NodeJS/metadata.json

```diff
@@ -11,5 +11,6 @@
         "path",
         "connection",
         "tableName"
-    ]
+    ],
+    "disabledInTryMode": false
```

Comment:
That makes sense
@fashaikh can you inverse the flag. This way we would also have a smaller change.